<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskWidgetClass</class>
 <widget class="QWidget" name="TaskWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>61</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>TaskWidget</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="taskWidgetHorizontalLayout">
    <item>
     <widget class="QCheckBox" name="markCompleteCheckBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string extracomment="Check to mark the task completed"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="taskLineEdit"/>
    </item>
    <item>
     <widget class="QToolButton" name="synchToolButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string extracomment="Synchronize this task"/>
      </property>
      <property name="text">
       <string>Synch</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="deleteToolButton">
      <property name="toolTip">
       <string extracomment="Delete this task"/>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
